src/lib/util/util.ts

```
@@ -1,12 +1,16 @@
     QuaverChannels,
     QuaverClient,
     QuaverPlayer,
     QuaverSong,
     SettingsPage,
+    SettingsPageFormatOptions,
+    SettingsPageGenericOptions,
     SettingsPageOptions,
+    SettingsPagePremiumOptions,
+    TopLevelComponentBuilders,
     WhitelistedFeatures,
 } from '#src/lib/util/common.d.js';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)